Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -297,6 +297,8 @@ var ScrollView = React.createClass({
     /**
      * A RefreshControl component, used to provide pull-to-refresh
      * functionality for the ScrollView.
+     *
+     * See [RefreshControl](/react-native/docs/refreshcontrol.html).
```

Comment:
I made it relative so it works when running the site locally but I agree that being able to use it from the source is more useful. I'll change it.